### PR TITLE
feat(e2s): add RoleTester service account key for mainnet part 1

### PR DESCRIPTION
**Pull request by kc1116**
This PR adds a RoleTester key for mainnet so that we can submit xmsgs to mainnet via a script. 

issue: #2275

### CODE REVIEW

**Comment by kc1116** [comment]
Currently the key generation tool assumes that all keys being created for a application service. This PR adds the `Service` key type so that we can generate keys for utility purposes.

```
 go run e2e/main.go key-create -f e2e/manifests/mainnet.toml --name=tester --type=service
```
We generated a new _service_ key for mainnet and mapped it to the RoleTester. Now we can access this key in the e2e app the same way we access RoleTester for other networks.

**Review by chmllr — APPROVED** [review]
Code LGTM, can't validate the logic yet.

File: e2e/app/eoa/static.go [hunk 1]
```diff
@@ -46,5 +46,6 @@ var statics = map[netconf.ID][]Account{
 		remote("0x9496Bf1Bd2Fa5BCba72062cC781cC97eA6930A13", RoleDeployer),
 		secret("0xfD62020Cee216Dc543E29752058Ee9f60f7D9Ff9", RoleMonitor),
 		secret("0x6191442101086253A636aecBCC870e4778490AaB", RoleRelayer),
+		secret("0x84aC7eeC1e035Fa13a7DEBD5f6da69B82CBDC5B5", RoleTester),
```
Comment (sideninja) [inline e2e/app/eoa/static.go:49]: wouldn't it be better for this account to be remote type, managed by Fireblocks?

File: e2e/app/key/key.go [hunk 2]
```diff
@@ -22,14 +22,15 @@ const (
 	P2PConsensus Type = "p2p_consensus"
 	P2PExecution Type = "p2p_execution"
 	EOA          Type = "eoa"
+	Service      Type = "service"
```
Comment (corverroos) [inline e2e/app/key/key.go:25]: no need for a new type, this is an eoa

File: e2e/manifests/mainnet.toml [hunk 3]
```diff
@@ -4,6 +4,9 @@ public_chains = ["ethereum","arbitrum_one","base","optimism"]
 multi_omni_evms = true
 prometheus   = true
 
+[keys.tester]
```
Comment (corverroos) [inline e2e/manifests/mainnet.toml:7]: don't think this is the way to go, it deplicates whatever is in the eoa package. Follow the same patterns as relayer and monitor keys.

**Review by corverroos — CHANGES_REQUESTED** [review]
Blocking review since christiaan aproved and this is the way to go

**Comment by kc1116** [comment]
Did not have to do.

**Pull request closed without merging**